ci/test/00_setup_env_arm.sh

```diff
@@ -17,6 +17,7 @@ if [ -n "$QEMU_USER_CMD" ]; then
   export PACKAGES="$PACKAGES qemu-user"
 fi
 # Use debian to avoid 404 apt errors when cross compiling
+export DOCKER_NAME=arm_linux_gnueabihf
```

Comment:
```suggestion
export DOCKER_NAME=ci_arm_linux
```

Reply:
Oh, and this line should be moved, because the above comment applies to `DOCKER_NAME_TAG`